### PR TITLE
Integrate database creation

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?
It integrates database creation

#### Ticket
https://trello.com/c/aV4Tk3Yf

#### Screenshot
In case a DB flavour is not selected
Screenshot from 2021-03-15 16-25-29

#### How to test this PR
1. Pull this branch
2. Click and drill into a project
3. click "databases" under the sidebar
4. Click the create button on the top right-hand corner to display a view from which to create a database.
5. Select a database flavour from the dropdown and click the create button.

### CODE REVIEW

**Review by MutegekiHenry — COMMENTED** [review]
The create works as expected however, the age seems wrong since newly-created dbs show 3hrs age. @neelxie could you look into this.

**Comment by neelxie** [comment]
> The create works as expected however, the age seems wrong since newly-created dbs show 3hrs age. @neelxie could you look into this.

@MutegekiHenry I raised this issue on Slack and I believe I will sort this in a different PR!

**Review by MutegekiHenry — APPROVED** [review]

File: src/components/CreateDatabase/index.jsx [hunk 1]
```diff
@@ -1,64 +1,161 @@
...
+import Feedback from '../Feedback';
+import createDatabase, { clearDatabaseCreateState } from '../../redux/actions/createDatabase';
 import './CreateDatabase.css';
 
 const flavours = [
-  { name: 'MYSQL' },
-  { name: 'PostgreSQL' },
-  { name: 'MongoDB' },
-  { name: 'MariaDB' }
+  { name: 'MYSQL', id: 1, value: 'MYSQL' }
```
Comment (kawerewagaba) [inline src/components/CreateDatabase/index.jsx:14]: we shall review this structure later

**Review by kawerewagaba — APPROVED** [review]
lgtm

**Pull request merged**